benchmarks/ai.html

```diff
@@ -67,13 +68,15 @@
 import Color from "../src/color.js";
 import srgb from "../src/spaces/srgb.js";
 import oklch from "../src/spaces/oklch.js";
-import "../src/deltaE/deltaE2000.js";
+import to from "../src/to.js";
+import inGamut from "../src/inGamut.js";
+import serialize from "../src/serialize.js";
 
 test("Color.js", colorjs_canvas, {
-	create: (l, c, h) => new Color('oklch', [l, c, h]),
-	to_srgb: color => color.to("srgb"),
-	in: color_srgb => color_srgb.inGamut(undefined, {epsilon: 0}),
-	str: color_srgb => color_srgb.toString()
+	create: (l, c, h) => ({space: oklch, coords: [l, c, h]}),
+	to_srgb: color => to(color, "srgb"),
```

Comment:
Does `srgb` color space register in `to()`?

We can do `to(color, srgb)` to avoid registration.